rasa/core/utils.py

```diff
@@ -19,8 +19,6 @@
 from rasa.constants import DEFAULT_SANIC_WORKERS, ENV_SANIC_WORKERS
 from rasa.shared.constants import DEFAULT_ENDPOINTS_PATH
 
-# backwards compatibility 1.0.x
-# noinspection PyUnresolvedReferences
```

Comment:
refered to [an import that no longer exists](https://github.com/RasaHQ/rasa/commit/07e2b5def944d146c5786f6d35ea23313c55ce04#diff-9c3ef27acfec367531f04a89510310550385f4d89daa83a27df0412a1c23d793R23-R27)